Networking/Networking/Remote/GenerativeContentRemote.swift

```diff
@@ -219,13 +219,20 @@ private extension GenerativeContentRemote {
                            categories: [ProductCategory],
                            tags: [ProductTag],
                            token: JWToken) async throws -> AIProduct {
-        let input = [
+        var inputComponents = [
             "You are a WooCommerce SEO and marketing expert, perform in-depth research about the product " +
             "using the provided name, keywords and tone, and give your response in the below JSON format.",
-            "name: ```\(productName)```",
             "keywords: ```\(keywords)```",
             "tone: ```\(tone)```",
-        ].joined(separator: "\n")
+        ]
+
+        // Name will be added only if `productName` is available.
+        // TODO: this code related to `productName` can be removed after releasing the new product creation with AI flow.
```

Comment:
Suggestion: Adding the GitHub issue number to the TODO will make it easier to track.